### PR TITLE
chore: update cooperation page & remove blue book

### PR DESCRIPTION
**Pull request by GuangmingLuo**
#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: content/en/cooperation/_index.md [hunk 1]
```diff
@@ -6,14 +6,7 @@ menu:
...
 ---
 
-{{< blocks/cover title="谁在使用CloudWeGo" image_anchor="bottom" height="min" >}}
-
-<p>
-CloudWeGo 项目介绍小蓝书 &nbsp&nbsp
-<a id="file_download_bluebook" href="https://github.com/cloudwego/community/raw/main/CloudWeGo_BlueBook_Project_Introduction.pdf"><i class="fas fa-download"></i></a>
-</p>
-
-<p class="lead mt-5">CloudWeGo 用户实践案例分享</p>
```
Comment (rogerogers) [inline content/en/cooperation/_index.md:16]: 这句话不留了吗，看着页面好奇怪

Comment (rogerogers) [reply content/en/cooperation/_index.md:16]: image

Comment (GuangmingLuo) [reply content/en/cooperation/_index.md:16]: 页面 Title 就用 CloudWeGo Enterprise Use Case 这个就行，多了也冗余。感觉这个页面未来还是要继续重构优化

Comment (rogerogers) [reply content/en/cooperation/_index.md:16]: OK

**Review by rogerogers — APPROVED** [review]

**Pull request merged**